src/lambdas/initiateAthenaQuery/handler.spec.ts

```diff
@@ -14,26 +15,37 @@ jest.mock('../../sharedServices/zendesk/updateZendeskTicket', () => ({
 jest.mock('../../sharedServices/dynamoDB/dynamoDBGet', () => ({
   getQueryByZendeskId: jest.fn()
 }))
+jest.mock('../../sharedServices/athena/createQuerySql', () => ({
+  createQuerySql: jest.fn()
+}))
 
 const mockConfirmAthenaTable = confirmAthenaTable as jest.Mock<
   Promise<ConfirmAthenaTableResult>
 >
 const mockUpdateZendeskTicket = updateZendeskTicketById as jest.Mock
 const mockGetQueryByZendeskId = getQueryByZendeskId as jest.Mock
+const mockCreateQuerySql = createQuerySql as jest.Mock
 
 describe('initiate athena query handler', () => {
   beforeEach(() => {
     mockConfirmAthenaTable.mockReset()
+    mockCreateQuerySql.mockReset()
```

Comment:
You can reset all your mocks in one go with `jest.resetMocks()`, a tip I got from Mabon!

Reply:
Thanks